docs/references/nextjs/overview.mdx

```diff
@@ -5,20 +5,28 @@ description: Learn how to use Clerk to quickly and easily add secure authenticat
 
 # Overview
 
-Clerk makes it simple to add authentication to your Next.js application. Looking to get started? Check out our [getting started guide](/docs/quickstarts/nextjs). If you want to learn more about the capabilities and methods available from Clerk's Next.js SDK, check out the reference content below.
+Clerk makes it simple to add authentication to your Next.js application. This documentation covers the capabilities and methods available from Clerk's Next.js SDK.
 
-## `authMiddleware()`
+## Guides
+
+- [Add custom sign up and sign in pages](/docs/references/nextjs/custom-signup-signin-pages)
+- [Integrate Clerk into your app with tRPC](/docs/references/nextjs/trpc)
+
+## References
```

Comment:
I feel like this order is confusing. The first sentence of content mentions auth() and getAuth() out of the blue. 

I feel like the following order would make more sense to a new customer:

- App Router
- Pages Router
- Client-side helps 
- Auth Object and authMiddleware() in either order to finish

Reply:
This is how it was organized before - I agree with this!